src/index.html

```diff
@@ -21,7 +21,335 @@
+            alt="MOYO-logo"
+          />
+        </a>
+      </div>
+
+      <nav class="catalog__nav">
+        <ul class="catalog__nav-list">
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link catalog__nav-link--is-active"
+              href="#apple"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#samsung"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#smartphones"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#laptops & computers"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#gadgets"
+            >
+              gadgets
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#tablets"
+            >
+              tablets
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#photo"
+            >
+              photo
+            </a>
+          </li>
+
+          <li class="catalog__nav-item">
+            <a
+              class="catalog__nav-link"
+              href="#video"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog__product-card">
+      <section
+        class="catalog__element"
+        data-qa="card"
+      >
+        <div class="catalog__img"></div>
+        <article class="catalog__info">
+          <p class="catalog__name">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </p>
+          <p class="catalog__code">Product code: 195434</p>
+          <article class="catalog__rating">
+            <div class="catalog__star"></div>
+            <div class="catalog__star"></div>
+            <div class="catalog__star"></div>
+            <div class="catalog__star"></div>
+            <div class="catalog__star"></div>
+            <p class="catalog__reviews">Reviews: 5</p>
+          </article>
+          <div class="catalog__price">
+            <p class="catalog__price-text">Price:</p>
+            <p class="catalog__price-numbers">$2,199</p>
+          </div>
+          <button
+            class="catalog__button"
+            data-qa="card-hover"
+          >
+            BUY
+          </button>
```

Comment:
Buttons should have a `type` attribute. If this button is submitting a form, it should have `type="submit"`. If it's just a button for a JavaScript action, it should be `type="button"`.